src/Http/Controllers/AppController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Whilesmart\LaravelAppAuthentication\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Str;
+use Illuminate\Validation\Rule;
+use Laravel\Passport\ClientRepository;
+use Whilesmart\LaravelAppAuthentication\Http\Responses\Helper;
+use Whilesmart\LaravelAppAuthentication\Models\App;
+use Whilesmart\LaravelAppAuthentication\Transformers\AppTransformer;
+
+class AppController extends Controller
+{
+    use Helper;
+
+    private ClientRepository $clientRepository;
+
+    /**
+     * AppController constructor.
+     */
+    public function __construct(ClientRepository $clientRepository)
+    {
+        $this->clientRepository = $clientRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return array|Response
+     */
+    public function index(Request $request)
+    {
+        if (isset($request->with)) {
+            Validator::make(
+                [
+                    'with' => $request->with,
+                ],
+                array_filter([
+                    'with' => ['sometimes', 'array', Rule::in((new AppTransformer)->getAvailableIncludes())],
+                ])
+            )->validate();
+        }
+        $perPage = $request->query('perPage', 10);
+        $user = $request->user();
+
+        $paginator = App::where('user_id', $user->id)->paginate($perPage);
```

Comment:
Apps can belong to a team, let's issue this for the future when the teams package is set.

Reply:
ok

Reply:
No issue created here.

Reply:
Done #3